### ISSUE TITLE
[BUG] Notification repeats indefinitely

### ISSUE THREAD

**Issue by SwingBling** [tomc128/noterly#20]
I am on the latest version of Noterly as of 2023-11-19 (1.4.2) on my Sansung Galaxy S10e running Android 12.
In my Noterly app settings, my snooze duration is set to 1 hour.
My task is scheduled and repeats daily. I get the notification for the task, but ignore it for a couple of hours, then press snooze, and then ignore it for a couple of hours again after the notification reappears. Then I actually complete the task and mark it as done in the notification. After this, the notification immediately reappears, I press done again, it reappears again, etc., instead of being set to re-notify the next day. To fix the bug, I have to disable the task's "repeating" setting, save the changes, and then the notification stops reappearing. Then I have to re-enable it and save. Here is a video showing the issue:

https://youtu.be/YCO0fFymZPM

**Comment by tomc128** [comment]
Hey, thanks for reporting this and providing the details you have, its really helpful! Sorry for the late reply, too!

I currently have a bunch of exams to study for, and I haven't been working on Noterly much. As soon as I can though, I plan to return and I will look into this issue!

**Comment by fillwithjoy1** [comment]
Can you use adb logcat and dump the logs here by any chance?

**Comment by SwingBling** [comment]
I installed adb and used logcat, but I wasn't sure how to filter the output of only the Noterly logs. Here's a video showing how to simply replicate the issue using a fresh install of the app:

https://youtu.be/QT_80XmzDR0

**Comment by fillwithjoy1** [comment]
> I installed adb and used logcat, but I wasn't sure how to filter the output of only the Noterly logs. Here's a video showing how to simply replicate the issue using a fresh install of the app:
> 
> https://youtu.be/QT_80XmzDR0

You can just post the logs as is, I'll filter them

**Comment by fillwithjoy1** [comment]
> I installed adb and used logcat, but I wasn't sure how to filter the output of only the Noterly logs. Here's a video showing how to simply replicate the issue using a fresh install of the app:
> 
> https://youtu.be/QT_80XmzDR0

Yep, I can reproduce it. Does this happen with any note you set to repeat and you snooze the daily reminder that comes up?

**Comment by SwingBling** [comment]
Yes, that is correct. It also happens when I set the snooze time for 1 hour instead of 1 minute, but I just made it 1 minute to make the video shorter.
-------- Original message --------From: Joy ***@***.***> Date: 1/26/24  3:55 AM  (GMT-05:00) To: tomc128/noterly ***@***.***> Cc: SwingBling ***@***.***>, Author ***@***.***> Subject: Re: [tomc128/noterly] [BUG] Notification repeats indefinitely (Issue #20) 

I installed adb and used logcat, but I wasn't sure how to filter the output of only the Noterly logs. Here's a video showing how to simply replicate the issue using a fresh install of the app:
https://youtu.be/QT_80XmzDR0

Yep, I can reproduce it. Does this happen with any note you set to repeat and you snooze the daily reminder that comes up?

**Comment by fillwithjoy1** [comment]
What an oddly specific case scenario, thanks for giving that info. I'll check when I have time but this should be enough for tomc128 to also investigate into the issue as well

**Comment by nvllz** [comment]
@tomc128 are you still using the app? I see a lot of appreciation messages for Noterly, but this bug and disappearing notifications is like a daily thing for me. Any idea why the notifications are so buggy? I love the app but this kind of unreliability is killing it.
